Neos.Flow/Classes/Http/Client/Browser.php

```diff
@@ -138,11 +140,14 @@ public function request($uri, $method = 'GET', array $arguments = [], array $fil
             throw new \InvalidArgumentException('$uri must be a URI object or a valid string representation of a URI.', 1333443624);
         }
 
-        $request = Request::create($uri, $method, $arguments, $files, $server);
-
-        if ($content !== null) {
-            $request->setContent($content);
+        $request = new ServerRequest($method, $uri, [], $content, '1.1', $server);
```

Comment:
Having a factory method for "our" needs is preferable I think. Doesn't need to be the old `Request` class, but I don't see a bunch of reasons why not either.

Reply:
Sure!